framework/src/org/apache/cordova/CordovaActivity.java

```diff
@@ -130,6 +147,18 @@ public void onCreate(Bundle savedInstanceState) {
             cordovaInterface.restoreInstanceState(savedInstanceState);
         }
     }
+
+    private void setSystemUiVisibilityMode(Window window) {
```

Comment:
If you're going to pass in the window, then you should make this method static.
If you're going to call it "set...Mode", then it should accept the mode as a parameter.
Really though, hoping you don't need the listener so that you don't need a helper at all.